src/integration/prepareApiReferences.js

```diff
@@ -49,15 +49,17 @@ function prepareApiReferences(registry, config, next) {
 
         // create API reference file
         createApiReferenceMainFile(apiReferenceSource, apiReferenceNewFile, apiReferenceContent, name),
-        createApiReferenceMainFile(apiReferenceSourceInternal, apiReferenceNewFileInternal, apiReferenceContentInternal, nameInternal),
+        createApiReferenceMainFile(apiReferenceSourceInternal, apiReferenceNewFileInternal, apiReferenceContentInternal, nameInternal)
 
+        // *** temporarily disabled ***
```

Comment:
I do not agree. It is clearly described: `create JS Client out of the RAML files` or `zip JS Client to client.zip archive` and it can be easily found. Still, information that something is temporarily disabled is quite important.